File: cinema_app/lib/data/models/age_restriction.dart

class AgeRestriction {
  String id;
  String name;
  String description;
  String abbreviation;
  bool status;

  AgeRestriction(
      {this.id = "", this.name = "", this.status = false, this.description = "", this.abbreviation=""});

  AgeRestriction.fromJson(Map<String, dynamic> json)
      : id = json["id"]??"",
        name = json["name"]??"",
        description = json["description"] ?? "",
        abbreviation=json["abbreviation"]??"",
        status = json["status"] ?? false;
}

File: Cinemabe/Cinema/Data/Models/AgeRestriction.cs
namespace Cinema.Data.Models
{
    public class AgeRestriction
    {
        public Guid Id { get; set; }
        public string Name { get; set; }
        public string Description { get; set; }
        public string Abbreviation { get; set; }
        public bool Status { get; set; }
    }
}
